### DISCUSSION TITLE
Cats Effect 3 migration

### DISCUSSION THREAD

**Discussion by tomasherman** [datadog4s/datadog4s#358] [General]
Hello,

I wonder if any of the users are interested in early version of this library based on Cats-Effect3? The issue is, that http4s is not migrated at this point. So in order to get out version of datadog4s, we would temporarily need to disable the builds of http4s module and make some other small tweaks. It's doable, but i wonder if it's necessary to rush this out. My suspicion is that http4s is one of the reasons why people use datadog4s.

BTW in case anyone interested, the branch with http4s disabled and everything migrated to CE3 is already prepared, it's just a question of whether this step is worth it (#346).

Thoughts? :) Thanks

**Comment by Body87** [comment]


**Reply by tomasherman** [reply]
Hi, not sure what do you mean :)

**Comment by tomasherman** [comment]
No longer relevant
